django_multitenant/tests/test_models.py

```diff
@@ -38,21 +40,24 @@ def test_filter_without_joins_on_tenant_id_not_pk(self):
         unset_current_tenant()
 
     def test_select_tenant(self):
+
         from .models import Project
 
+        # pylint: disable=pointless-statement
```

Comment:
self.projects seems pointless but when it is accessed sql statements are being executed and data is fetched